shopping_mall/backend/ai/seed_rag.py

```
@@ -34,6 +34,22 @@
+
+    pypdf가 글자 사이에 공백을 삽입하는 경우 (예: '제 1 조 ( 주문  가능  시간 )')
+    조/장 패턴과 줄 내부 연속 공백을 정규화하여 청킹 정규식이 작동하게 한다.
+    """
+    # 줄 단위로 연속 공백 → 단일 공백
+    lines = [re.sub(r"[ \t]{2,}", " ", line.strip()) for line in text.split("\n")]
+    text = "\n".join(line for line in lines if line)
+    # "제 N 조(...)" → "제N조(...)"
+    text = re.sub(r"제\s+(\d+)\s+조\s*\(", r"제\1조(", text)
+    # "제 N 장" → "제N장"
+    text = re.sub(r"제\s+(\d+)\s+장", r"제\1장", text)
+    return text
```

Ruff (0.15.10)

[warning] Unnecessary assignment to `text` before `return` statement

Remove unnecessary assignment

(RET504)